libs/shared/interfaces/src/lib/domain/git-application.entity.ts

```diff
@@ -37,4 +38,8 @@ export interface GitApplicationEntity extends Application {
     loadingStatus: LoadingStatus
     default_settings?: ApplicationAdvancedSettings
   }
+  logs?: {
```

Comment:
Did not we say that finally the log are in the context and not in the store?

Reply:
It's an oversight, I'll remove it thank you!